### ISSUE TITLE
Pre Launch Checklist

### ISSUE THREAD

**Issue by rbarman** [PinkShirtBoys/ThinkTank#4]
Below are a list of things that must be done before launching to ensure that our beta users have a solid starting experience and that we can continue to develop new changes while the beta period is in place.

Please add more if you think of any and create a separate issue for it if necessary. 
1. Test and Production databases
   - There needs to be a test db where we can make modifications and test code without disrupting what the users see. Users should only be accessing information that is visible on the production database. 
2. Email Verification checks
   - Some emails get an invalid verification link. This makes it impossible for some users to sign up
   - Issue has already been [made](https://github.com/rbarman/ThinkTank/issues/2)
3. Basic browser checks
   - Check basic behavior and views are working in modern browsers
4. Sample debates 
   - Add some sample debates that users can look through to get a feel for what our platform offers 
5. Help / Contact Us form
   - Some form where users can enter in issues or problems they have faced and we can review them.
6. Create Parse security rules
   - Someone can get our Parse API keys by inspecting our web app
   - Need to create Parse security rules to prevent unwanted api manipulation

**Comment by pranavpunjabi** [comment]
1. Test and Production Databases
   
    There are a lot of people struggling with the same issue of having two different DB's and the only solution that comes out of the whole discussion is to build a separate parse app for development server and clone the DB.
   https://parse.com/questions/does-parse-support-development-and-production-or-do-you-have-to-have-2-subscriptions
2. Email Verification checks
   - I have tried two valid emails both receiving valid links. Will also try with fake/random generated     email ids
   - How do you use email ids such as a@a.com, b@b.com, c@c.com for testing? They don't seem verified.
3. Basic browser checks
   - [x] Google Chrome 
   - [X] Firefox 
   - [x] Edge 
   - [ ] Safari 
4. Sample Debates
   Added a few. Will be adding a few more over the time.
5. Help/ContactUs Form
   Planning on going for a survey rather than an in-application form
6. Create Parse Security rules
   [Remove Sensistive Data](https://help.github.com/articles/remove-sensitive-data/)

**Comment by rbarman** [comment]
I am able to `a@a.com` type emails because they have an `undefined` value for their `emailVerified` fields. When a user signs in via `SignInCtrl.js`,  there is no check for an `undefined` value

Also added another pre-launch task
